lib/cbus_elixir/app/meetings.ex

```diff
@@ -0,0 +1,31 @@
+defmodule CbusElixir.App.Meetings do
+  @moduledoc """
+  Domain operations for meetings.
+  """
+
+  import Ecto.Query, warn: false
+  alias CbusElixir.Repo
+
+  alias CbusElixir.App.Meeting
+
+  @doc """
+  Returns the next 5 upcoming meetings
+  """
+  def upcoming_meetings_for(date) do
+    Meeting
+    |> Meeting.happening_after(date)
+    |> limit(5)
+    |> Repo.all
+  end
+
+  @doc """
+  Returns the next upcoming meeting including speakers
+  """
+  def next_meeting(date) do
+    Meeting
+    |> Meeting.happening_after(date)
+    |> first
```

Comment:
This isn't an ordered query. You're going to want to sort here or in `happening_after`

Reply:
`happening_after` includes an `order_by`.